src/mac/hmac/hmac_file.c

```diff
@@ -63,12 +59,15 @@ int hmac_file(int hash, const char *fname,
       goto LBL_ERR;
    }
 
-   /* process the file contents */
+   if ((err = hmac_init(&hmac, hash, key, keylen)) != CRYPT_OK) {
```

Comment:
why this? I liked it better before

... even if that means modifying all the others... :)